be1-go/channel/lao/mod.go

```diff
@@ -477,6 +504,15 @@ func (c *Channel) processRollCallObject(action string, msg message.Message) erro
 	return nil
 }
 
+func createChirpingChannel(publicKey string, channelID string, hub channel.HubFunctionalities,
+							general *generalChriping.Channel, log zerolog.Logger) {
+	chirpingChannelPath := channelID + "/" + "social" + "/" + publicKey
+	cha := chirp.NewChannel(chirpingChannelPath, publicKey, hub, general, log)
+	hub.RegisterNewChannel(chirpingChannelPath, &cha)
+	log.Info().Msgf("storing new channel chirp channel for : '%s', name of the channel : '%s' ", 
```

Comment:
`log.Info().Msgf("storing new chirp channel (%s) for: '%s'", publicKey, channelID)`